unity/Hello_World/Assets/Scenes/my_scene.unity

```diff
@@ -285,6 +285,7 @@ MonoBehaviour:
   m_Script: {fileID: 11500000, guid: e555917419c843047b1c560ed473073f, type: 3}
   m_Name: 
   m_EditorClassIdentifier: 
+  topicName: PVFrames
```

Comment:
It looks like that gets pulled from `PVCameraCapture.cs` when you open the script object in the Unity inspector. I hadn't opened the inspector for that script so I guess it wasn't updated with the latest variables. I tested it and you can set those member variables from the inspector and they'll override what the script defaults them to. So in the picture below, the image topic name would be set to "test" instead of "PVFramesNV12".

![image](https://user-images.githubusercontent.com/92824826/163228614-2794de13-5546-439f-bf87-dfffb620ae7f.png)